app/features/agicash-db/database.ts

```diff
@@ -248,9 +248,9 @@ export const agicashDb = createClient<Database>(supabaseUrl, supabaseAnonKey, {
     schema: 'wallet',
   },
   realtime: {
-    logger: (kind: string, msg: string, data?: unknown) => {
+    logger: (kind: string, msg: unknown, data?: unknown) => {
       console.log(
-        `Realtime (${new Date().toISOString()}) -> ${kind}: ${msg}`,
+        `Realtime (${new Date().toISOString()}) -> ${kind}: ${typeof msg === 'string' ? msg : JSON.stringify(msg)}`,
```

Comment:
I changed this because I noticed that sometimes before the error happens I see this log `Realtime (2025-09-24T15:17:53.404Z) -> transport: [object Event] undefined` and I am hoping that I will be able to see what msg actually is if I stringify it like this